cmake/public/LoadHIP.cmake

```diff
@@ -46,4 +90,34 @@ set(CMAKE_MODULE_PATH ${HIP_PATH}/cmake ${CMAKE_MODULE_PATH})
 ADD_DEFINITIONS(-DNDEBUG)
 
 # Find the HIP Package
-FIND_PACKAGE(HIP 1.0 REQUIRED)
+FIND_PACKAGE(HIP 1.0)
+
+IF(HIP_FOUND)
```

Comment:
Why add the variable PYTORCH_FOUND_HIP? Shouldn't HIP_FOUND suffice?

PYTORCH_MIOPEN_LIBRARIES and PYTORCH_HIP_HCC_LIBRARIES can be renamed to "miopen_LIBRARIES" and "hip_hcc_LIBRARIES"

Reply:
@Jorghi12 Because this is a custom find, (at line 17) if HIP_PTH doesn't exist then it will simply return instead of going up to here. And overtime we likely will introduce more logic on top of HIP's own find.
Regarding to PYTORCH_MIOPEN_LIBRARIES and hip_hcc_LIBRARIES, see my two "TODO" comments at the bottom of these files.

Reply:
So, the obvious next question, is why are you returning immediately when `HIP_PATH` is set? :)

Reply:
@ezyang It's when HIP_PATH does not exist

Reply:
Since PYTORCH_MIOPEN_LIBRARIES / PYTORCH_HIP_HCC_LIBRARIES / PYTORCH_FOUND_HIP is being used by both frameworks, it's better to leave the names neutral (i.e. remove the PyTorch prefix).

Otherwise looks good!

Reply:
@Jorghi12 MIOPEN_LIBRARIES and HIP_HCC_LIBRARIES have been used by hip's own cmake files but there are some bugs in them that have been pointed out in my comments so we can not directly use these. And as the cmake files are merging together, and caffe2 has been merged into the pytorch repo, I think using PYTORCH as the prefix is ok?